### ISSUE TITLE
loading january 02 0001 save crashes game

### ISSUE THREAD

**Issue by kennyb** [MaFi-Games/Captain-of-Industry-issues#1407]
### Issue description

no mods were used.

```
The game failed to initialize likely due to a mod, see logs for more info.

System.MissingMethodException: Method not found: void Mafi.Core.Prototypes.Proto.SetAvailability(bool)
  at jOgQY3RGtH5fd9qQao.OBqe2IUAeSpOmlOQ4O+X6oonhL1IpoG9H0Sh52.tbKjk9l4qb (System.Boolean&  ) [0x001dd]  
  at jOgQY3RGtH5fd9qQao.OBqe2IUAeSpOmlOQ4O+X6oonhL1IpoG9H0Sh52.ygDj1mPbak () [0x00002]  
  at jOgQY3RGtH5fd9qQao.OBqe2IUAeSpOmlOQ4O.qWAa9mQMya[T] (System.Int32  , System.Object[]  , System.Object  , T&  ) [0x00838]  
  at jOgQY3RGtH5fd9qQao.OBqe2IUAeSpOmlOQ4O.TyOaFSuuHy (System.Int32  , System.Object[]  , System.Object  ) [0x00002]  
  at Mafi.Supporter.SupporterMod.RegisterPrototypes (Mafi.Core.Mods.ProtoRegistrator registrator) [0x00016]  
  at Mafi.Core.Game.GameBuilder.RegisterModsPrototypes (Mafi.Collections.ImmutableCollections.ImmutableArray`1[T] mods, Mafi.Core.Mods.ProtoRegistrator protoRegistrator, System.Boolean noThrow) [0x00040]  
  at Mafi.Core.Game.GameBuilder.RegisterModsPrototypes (Mafi.Collections.ImmutableCollections.ImmutableArray`1[T] mods, Mafi.Collections.ImmutableCollections.ImmutableArray`1[T] configs, Mafi.Core.Prototypes.ProtosDb protosDb, System.Action`2[T1,T2] protosDbRegFnBeforeMods, System.Action`2[T1,T2] protosDbRegFnAfterMods, System.Boolean noThrow) [0x00013]  
  at Mafi.Core.Game.GameBuilder+<BuildLoadedGameTimeSliced>d__4.MoveNext () [0x001ca]  
  at Mafi.Unity.GamePlayScene+<loadGame>d__17.MoveNext () [0x00205]  
  at Mafi.Unity.GamePlayScene+<Initialize>d__15.MoveNext () [0x000a5]  
  at Mafi.Unity.Main.stepSceneInitialization () [0x000d4] 
```

****

### Reproduction steps

1. start new game on admiral
2. immediately pause after closing tutorial panel
3. save the game
4. (optional) make a mistake on something you wanted to make
5. load the save game

### Game version and build number

v0.6.0 b309

### Operating system

Windows

### Attachments

[Save 2024-04-08, 18-53-57.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/14912177/Save.2024-04-08.18-53-57.zip)

**Comment by zuff-coi** [comment]
Is the save you attached the one you cannot load? 

I was able to load that fine and following your steps was not able to reproduce the issue. If you have a save you cannot load please share it!

**Comment by kennyb** [comment]
When reading #1410, I noticed that the mod that was failing for them is `Mafi.Supporter.SupporterMod`, so I tried renaming my DLCs folder to something else, and sure enough it loads without a problem. renaming the DLCs folder (which contains the supporter mod) gives me the same error as they have. so, this is a dup of that one (closing this one). the solution is to delete/rename the DLCs folder so it doesn't load the supporter's mod.
thanks!
